client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import {
+	setCustomsItemDescription,
+	setCustomsItemTariffNumber,
+	setCustomsItemOriginCountry,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import CountryDropdown from 'woocommerce/woocommerce-services/components/country-dropdown';
+
+const TariffCodeTitle = localize( ( { translate } ) =>
+	<span>{ translate( 'Tariff Code' ) } (<a href="https://hts.usitc.gov/"
+		target="_blank" rel="noopener noreferrer">{ translate( 'look up' ) }</a>)</span>
+);
+
+const ItemRow = ( props ) => {
+	const {
+		errors,
+		packageId,
+		productId,
+		translate,
+		description,
+		defaultDescription,
+		tariffNumber,
+		originCountry,
+		countriesData,
+	} = props;
+
+	return <div className="customs-step__item-row">
```

Comment:
Any reason not to use `return ()`?

Reply:
Yeh, mostly because that's the way most files I see in Calypso do it, so I stumbled over this. But it's a style nitpick that's better handled by a linter, so it's all good if you don't want to change it :)